### ISSUE TITLE
Use standard NSIDC open-source README and other elements from template repo

### ISSUE THREAD

**Issue by MattF-NSIDC** [nsidc/nsidc_open_science#9]
We have a standard format for license, code of conduct, level of service, and more available at https://github.com/nsidc/template

**Comment by andypbarrett** [comment]
OK.  Happy to use that.  I will update readme accordingly.  We should maybe have a how-to on setting up a github for a new project using these standard resources.

**Comment by MattF-NSIDC** [comment]
Agree! We could create a `how-tos` directory in this repo and add markdown files for various tasks? Or we could link to the git-training project for how-tos?
